images/jdk/main.tf

```
@@ -11,11 +11,12 @@ variable "target_repository" {
 module "latest-config" { source = "./config" }
 
 module "latest" {
-  source            = "../../tflib/publisher"
-  name              = basename(path.module)
-  target_repository = var.target_repository
-  config            = module.latest-config.config
-  build-dev         = true
+  source             = "../../tflib/publisher"
+  name               = basename(path.module)
+  target_repository  = var.target_repository
+  config             = module.latest-config.config
+  build-dev          = true
+  extra_dev_packages = each.key != "8" ? ["openjdk-${each.key}-jmods"] : []
```

Since this is always building the latest version, we don't need to take a versioned approach (or filter out OpenJDK 8):

```suggestion
  extra_dev_packages = ["openjdk-jmods"]
```

---

Yeah, I was just being silly. I can't figure out how to just use `openjdk-mods` and need to use `openjdk-22-mods` though.